docs/receiving/using-app-portal/adding-endpoints.mdx

```diff
@@ -24,3 +24,27 @@ full descriptions of each endpoint as well as all the fields they send.
 If you don't specify any event types, by default, your endpoint will receive all events, regardless of type.
 This can be helpful for getting started and for testing, but we recommend changing this to a subset later on
 to avoid receiving unexpected messages.
+
+### Configuring mTLS
+
+Svix now supports Mutual TLS (mTLS), per endpoint. At this time this feature is only configurable via API.
+
+- `PUT $SVIX_ROOT/api/v1/app/APP/endpoint/ENDPOINT/mtls` to set the configuration options.
```

Comment:
Doesn't need to be now, just putting it for posterity, but let's follow the format in the rest of the docs with how we show URLs, endpoint IDs in URLS, etc.

All the `$TOKEN` and `$SVIX_ROOT` is very confusing.